### ISSUE TITLE
Way to find min+max for ipv6 subnet

### ISSUE THREAD

**Issue by simPod** [ClickHouse/ClickHouse#55241]
_My main goal is to find out whether /24 subnet is in /19 subnet_

Consider `2001:268:0:0:0:0:0:0/32`

Since there are almost no range functions in ClickHouse, I'm trying to extract min and max ips so I can then use `isIPAddressInRange()` function.

I was able to make it work for ipv4 using something similar https://github.com/ClickHouse/ClickHouse/issues/247#issuecomment-266749237

Is there a way to do something similar for ipv6?

I kinda found a subnet min for `2001:1260:230:0:0:0:0:0/45` as `cutIPv6(IPv6StringToNum('2001:1260:230:0:0:0:0:0'), 0, 0) as min` but I'm not sure how to find max.

Basically I tried `reinterpretAsUInt256(IPv6StringToNum('2001:1260:230:0:0:0:0:0')) + 524288` but did not find a way to convert uint256 back to fixedstring(16).

**Comment by simPod** [comment]
Ah, sure, thanks!
